chunkator/__init__.py

```diff
@@ -28,7 +36,18 @@ def chunkator(source_qs, chunk_size, query_log=None):
         if "pk" not in source_qs._fields:
             raise MissingPkFieldException("The values() call must include the `pk` field")  # noqa
 
-    source_qs = source_qs.order_by('pk')
+    if order_by_field != 'pk':
+        field = source_qs.model._meta.get_field(order_by_field)
+        if not field.primary_key:
```

Comment:
As discussed IRL: it is possible to have a field with `unique=True` and `null=True`. So we chosed not to support unique fields which are not primary keys.